keyvault/client/secrets/README.md

```diff
@@ -0,0 +1,252 @@
+# Azure Key Vault Secret client library for Java
+Azure Key Vault is a tool for securely storing and accessing secrets. A secret is anything that you want to tightly control access to, such as API keys or passwords. A vault is logical group of secrets.
+Secret client library allows you to securely store and tightly control access to tokens, passwords, API keys, and other secrets. THe library offers operations to create, retrieve, update, delete, purge, backup, restore and and list the secrets.
+
+Use the secret client library to create and manage secrets.
+
+[Source code][source_code] | [Package (Maven)][package] | [API reference documentation][api_documentation] | [Product documentation][azkeyvault_docs]
+
+## Getting started
+
+### Prerequisites
+
+- [Java Development Kit (JDK)][jdk] with version 8 or above
+- [Azure Subscription][azure_subscription]
+- [Azure KeyVault][azure_keyvault]
+
+### Adding the package to your product
+
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-keyvault-secrets</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+</dependency>
+```
+
+### Create a Secret in the Azure Key Vault.
+
+To create an Azure Key Vault Store you can use the Azure Portal or [Azure CLI][azure_cli].
+
+Create the Azure Key Vault:
+```Powershell
+az keyvault create --name <keyvault-name> --resource-group <resource-group-name> --location eastus
+```
+
+After that, create the secret in the Azure Key Vault:
+```Powershell
+az keyvault secret set --vault-name <keyvault-name> --name "<secret-name>" --value "<secret-value>"
+```
```

Comment:
We'll need to this front-matter approved before we use it. I definitely agree we want something here to help people get started.

Reply:
I have creating/using an existing keyvault in the VaultClient readme's prerequisite [here](https://github.com/samvaity/azure-sdk-for-python/tree/keyvault-track2-readme/sdk/keyvault/azure-security-keyvault#prerequisites)

Do we want to do the same?
It also has an additional link which provides detailed steps to create keyvault and secret.